fe1-web/src/core/components/PoPButton.tsx

```diff
@@ -20,30 +20,36 @@ const styles = StyleSheet.create({
     borderColor: Color.inactive,
     backgroundColor: Color.inactive,
   } as ViewStyle,
-  negative: {
+  outline: {
+    borderColor: Color.accent,
+    backgroundColor: Color.contrast,
+  } as ViewStyle,
+  negativeBorder: {
     borderColor: Color.contrast,
   } as ViewStyle,
-  disabledNegative: {
+  disabledNegativeBorder: {
     borderColor: Color.inactive,
     backgroundColor: Color.inactive,
   } as ViewStyle,
 });
 
 const PoPButton = (props: IPropTypes) => {
-  const { onPress, disabled, children, negative, testID } = props;
+  const { onPress, disabled, children, outline, negativeBorder, margin, testID } = props;
 
   const viewStyles = [styles.button];
-  if (negative && disabled) {
-    viewStyles.push(styles.disabledNegative);
-  } else if (negative) {
-    viewStyles.push(styles.negative);
+  if (negativeBorder && disabled) {
+    viewStyles.push(styles.disabledNegativeBorder);
+  } else if (negativeBorder) {
+    viewStyles.push(styles.negativeBorder);
```

Comment:
I'm not sure about the use of "negative", is it "negative" as in "negative space" ? 
What's the connotation here ?

Reply:
Negative as in colored background / white foreground rather then black/colored text on white background. I think this is the term that refers to this?

Reply:
As discussed orally, we'll make a pass on this after the UI has been harmonized.